app/v1/connectors/CreateAndAmendEmploymentExpensesConnector.scala

```diff
@@ -29,15 +30,21 @@ import scala.concurrent.{ExecutionContext, Future}
 @Singleton
 class CreateAndAmendEmploymentExpensesConnector @Inject() (val http: HttpClient, val appConfig: AppConfig) extends BaseDownstreamConnector {
 
-  def amend(request: CreateAndAmendEmploymentExpensesRequest)(implicit
-      hc: HeaderCarrier,
-      ec: ExecutionContext,
-      correlationId: String): Future[DownstreamOutcome[Unit]] = {
+  def createAmendEmploymentExpenses(request: CreateAndAmendEmploymentExpensesRequest)(implicit
```

Comment:
could you make sure you're using the Scalafmt formatter in your project settings, rather than the intellij one? Then probably hit 'reformat' on all of these classes

(the above imports need reformatting either way!)